generator/sbpg/targets/resources/rust/sbp_messages_mod.rs

```diff
@@ -89,6 +99,8 @@ pub trait SbpMessage: WireFormat + Clone + Sized {
     fn friendly_name(&self) -> &'static str {
         ""
     }
+    /// Tells you if the message is invalid and may need to be special cased at certain points.
+    fn is_invalid(&self) -> bool;
```

Comment:
I hate to ask, but is it possible to rephase this as `is_valid` so that we don't have the double negative of `! is_invalid()` that means "valid"?